ansible/group_vars/all.yml

```diff
@@ -146,4 +146,3 @@ report_builder_add_on_email: sales@dimagi.com
 eula_change_email: eula-notifications@dimagi.com
 contact_email: info@dimagi.com
 soft_assert_email: commcarehq-ops+soft_asserts@dimagi.com
-daily_deploy_email: tech-announce-daily@dimagi.com
```

Comment:
I believe you can override this directly in the *`_public.yml` files, so you can just set it to `null` here and add `daily_deploy_email: tech-announce-daily@dimagi.com` in those three other files without changing `localsettings.py.j2`.